jp2_pc/Source/Trespass/video.cpp

```diff
@@ -176,7 +176,7 @@ void CVideoWnd::NextNonDirect()
     }
 
     m_pBuff->Unlock();
-    pSurface->Unlock(dds.lpSurface);
+    pSurface->Unlock(nullptr);
 }
```

Comment:
Here the interface has changed. We no longer have to specify the surface descriptor to unlock. Instead a rectangle (`LPRECT`) can be specified if the surface was only partially locked.